snakeware/overlay/usr/share/snakeware/startup.py

```diff
@@ -0,0 +1,66 @@
+import os
+import subprocess
+from shutil import which
+from suplemon.main import App
+
+
+THE_CODE_FILE = "/tmp/code.py"
+# Create an empty file to make sure that `list` and `run` do not crash. 
+if not os.path.isfile(THE_CODE_FILE):
+    with open(THE_CODE_FILE, "w") as fp:
+        fp.write("")
+THIS_PYTHON = "/usr/bin/python3"
+LESS_PRG = which("less")
+
+
+class Command:
+    """Defines a command which is run when repr(self) is evaluated."""
+
+    def run(self):
+        raise NotImplementedError
+
+    def __repr__(self):
+        """Execute the command."""
+        return self.run()
+
+
+class SnakeWMCommand(Command):
+    def run(self):
+        """Start SnakeWM."""
+        from snakewm.wm import SnakeWM
+
+        return SnakeWM().run()
+
+
+class EditCommand(Command):
+    """Start suplemon with this single file to edit."""
+
+    def __repr__(self):
```

Comment:
You should use `run` instead, because `__repr__` might get additional functionality later on.